.github/workflows/test-pool.yml

```
@@ -33,22 +33,20 @@ jobs:
         path: pool-simulation
         token: ${{ secrets.GITHUB_CI_BOT_TOKEN }}
 
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.10'
-
     - name: Generate requirements.txt
       run: |
         echo 'uvloop==0.15.2' > requirements.txt
         mkdir -p pool-simulation/reports
 
-    - name: Handle virtualenv
-      uses: syphar/restore-virtualenv@v1.1
-      id: venv-cache
+    - name: Set up Python
+      id: py310
```

```suggestion
      id: python
```

so that we don't forget/have to change this every time we bump the Python version